onadata/settings/base.py

```diff
@@ -702,33 +702,43 @@ def skip_suspicious_operations(record):
 ################################
 # MongoDB settings             #
 ################################
+if not (MONGO_DB_URL := env.str('MONGO_DB_URL', False)):
```

Comment:
Are we ok with Python 3.8+? https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions 

Fine with me, but noting the impact here.

Reply:
yeah, let's not do this. @noliveleger, i thought it was okay before, but Python 3.7 is a supported version through June 27, 2023. i will remove `:=` where we used it previously.

Reply:
nevermind… we've used this syntax since March of 2021
https://github.com/kobotoolbox/kobocat/commit/65f0daf2cba3e3719267568718b9935395826da1#diff-77c8b3554c67addf9515570c68b6d8ddd366484182f7ba3129db2df5a325769fR133